src/mission_execution/src/MissionExecutor.cpp

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+#include <memory>
+#include <exception>
+#include "mission_execution/MissionExecutor.hpp"
+
+using namespace std::chrono_literals;
+using nav2_tasks::TaskStatus;
+
+namespace mission_execution
+{
+
+MissionExecutor::MissionExecutor(const std::string & name)
+: nav2_tasks::ExecuteMissionTaskServer(name)
+{
+  RCLCPP_INFO(get_logger(), "MissionExecutor::MissionExecutor");

```

Comment:
Can we change these RCLCPP_INFO statements to something like "Starting Mission Executor", "Shutting down..." etc.

Reply:
Sure. These were just to ensure everything was created and wired up correctly. We can go back and make them more meaningful.